apps/meteor/server/email/IMAPInterceptor.ts

```diff
@@ -121,26 +121,19 @@ export class IMAPInterceptor extends EventEmitter {
 			this.stop();
 			return this.selfDisable();
 		}
+
 		if (this.backoff) {
 			clearTimeout(this.backoff);
 			this.backoffDurationMS = 3000;
 		}
-		const loop = async (): Promise<void> => {
-			logger.debug(`Reconnecting to ${this.config.user}: ${this.retries}`);
-			if (this.canRetry()) {
-				this.backoffDurationMS *= 2;
-				this.backoff = setTimeout(loop, this.backoffDurationMS);
-			} else {
-				logger.info(`IMAP reconnection failed on inbox ${this.config.user}`);
-				clearTimeout(this.backoff);
```

Comment:
these seem unrelated?

Reply:
it depends, its related because without that this pr will never pass since we dont handle imap errors.

but its not related because involve fixes, refactors whatever... because of that cherry-picked the commit
https://github.com/RocketChat/Rocket.Chat/pull/33855

as soon this one get merged, i will rebase and this changes will be dropped